src/tests/ftest/util/dmg_utils.py

```diff
@@ -922,6 +928,62 @@ def config_generate(self, access_points, num_engines=None, min_ssds=None,
 
         return yaml_data
 
+    def telemetry_metrics_list(self, host):
+        """List telemetry metrics.
+
+        Args:
+            host (str): Server host from which to obtain the metrics
+
+        Raises:
+            CommandFailure: if the dmg system query command fails.
+
+        Returns:
+            dict: dictionary of output in JSON format
+
+        """
+        return self._get_json_result(
+            ("telemetry", "metrics", "list"), host=host)
+
+    def telemetry_metrics_query(self, host, metrics=None):
+        """Query telemetry metrics.
+
+        Args:
+            host (str): Server host from which to obtain the metrics
+            metrics (str, None): Comma-separated list of metric names to query.
+                Defaults to None which will query all metric names.
+
+        Raises:
+            CommandFailure: if the dmg system query command fails.
+
+        Returns:
+            dict: dictionary of output in JSON format
+
+        """
+        # Sample output (metric="process_start_time_seconds"):
+        # {
+        # "response": {
+        #   "metric_sets": [
+        #     {
+        #       "name": "process_start_time_seconds",
+        #       "description": "Start time of the process since unix epoch in
+        #                       seconds.",
+        #       "type": 3,
```

Comment:
Just a note for myself, I'm noticing that the type value is unhelpful here. I created a ticket to change this to a string value of the enum.